.github/workflows/ci.yml

```diff
@@ -1,50 +1,42 @@
-# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
-# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+name: Continuous integration
 
-name: Node.js CI
-
-on:
-    push:
-        branches: [master]
-    pull_request:
```

Comment:
We NEED pull_request, we can't check jobs executed on a fork from a push event

Reply:
Done
See previous comment about renaming and tidying up